README.md

```diff
@@ -16,3 +16,5 @@ npm run dev
 `npm run prepare` installs the precommit hook, that will run Eslint and Prettier before you commit anything.
 
 `npm run dev` check [Graphiql](http://localhost:3000/graphiql) with your browser to test the GraphQL server.
+
+`npm install --save @supabase/supabase-js` installs necessary packages for Supabase database
```

Comment:
We don't need to explain how to install specific packages in the Readme, they should be installed automatically using npm install if they were added to package.json